### PR TITLE
Fix API aliases being orphaned

### PR DESCRIPTION
**Pull request by Eric-Arellano**
We have some attribute pages that are aliases, like `Sampler` being an alias for `SamplerV1`. These pages weren't showing up in the ToC (orphans) because of an oversight.

This PR also makes these improvements:

* Fixes our code for copying release note images to the correct location for Qiskit, as a continuation of https://github.com/Qiskit/documentation/pull/1914
* Uses `ApiType` rather than `string` in more places so that we get stronger typing
* Removes an unused arg, `$main`

### CODE REVIEW

File: docs/api/qiskit/release-notes/1.0.mdx [hunk 1]
```diff
@@ -587,7 +587,7 @@ This guide is meant to complement the release notes and have a targeted advice f
     qc.draw("mpl")
     ```
 
-    ![\_images/release\_notes-1.png](/images/api/qiskit/release_notes-1.png)
+    ![\_images/release\_notes-1.png](/images/api/qiskit/1.0/release_notes-1.png)
```
Comment (Eric-Arellano) [inline docs/api/qiskit/release-notes/1.0.mdx:590]: This was changed in https://github.com/Qiskit/documentation/pull/1914. I didn't think to regenerate 1.0

File: scripts/js/lib/api/generateToc.ts [hunk 2]
```diff
@@ -74,9 +74,15 @@ function getModulesAndItems(
...
-      result.meta.apiType === "exception",
+      result.meta.apiType &&
+      [
+        "class",
+        "function",
+        "exception",
+        "method",
+        "property",
+        "attribute",
+      ].includes(result.meta.apiType),
```
Comment (frankharkins) [inline scripts/js/lib/api/generateToc.ts:85]: Is this what makes the Sampler and Estimator appear in the TOC?

**Review by frankharkins — APPROVED** [review]
Thanks!

File: scripts/js/lib/api/generateToc.ts (hunk 2 above)
Comment (Eric-Arellano) [reply scripts/js/lib/api/generateToc.ts:85]: Yes, exactly. In the upcoming PR to add `..py:data::`, we'll add `data` here as well which fixes the remaining orphan pages we have in API docs 🚀

**Pull request merged**